### PR TITLE
twitter feed

### PR DESCRIPTION
**Pull request by rinchenlama0075**
Added twitter module on landing page.

### CODE REVIEW

File: _data/organizers.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+##file used as a trial##
```
Comment (DiegoAlbertoTorres) [inline _data/organizers.yml]: Plx remove this file.

File: _includes/posts.html [hunk 2]
```diff
@@ -3,35 +3,49 @@
...
-		<div class="content align-center div-center">
-                {% endif %}
-                {% if post.customurl %}
-			<h2><a href="{{ post.customurl}}" class="link">{{ post.title }}</a></h2>
-                {% else %}
-			<h2><a href="{{ post.url | relative_url }}" class="link">{{ post.title }}</a></h2>
-                {% endif %}
-			<p>{{ post.content }}</p>
-		</div>
+			{% else %}
```
Comment (DiegoAlbertoTorres) [inline _includes/posts.html]: Fix indentation for this else

File: _includes/posts.html [hunk 3]
```diff
@@ -3,35 +3,49 @@
...
+			<iframe width="100%" height="370" src={{ post.youtube }} frameborder="0"
+				allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+				allowfullscreen></iframe>
 			{% endif %}
 			{% if post.image %}
-				<img src="{% if site.featured-image-source %}{{ post.image | prepend: site.featured-image-source | absolute_url }}{% else %}{{ "" | absolute_url }}/assets/images/{{ post.image }}{% endif %}" alt="" />
+			<img src="{% if site.featured-image-source %}{{ post.image | prepend: site.featured-image-source | absolute_url }}{% else %}{{ "" | absolute_url }}/assets/images/{{ post.image }}{% endif %}"
+				alt="" />
 			{% endif %}
+			{% if post.twitter %}
```
Comment (DiegoAlbertoTorres) [inline _includes/posts.html]: Use elsif here and in `if post.image`

File: build/README.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
+## global.pydata.org
```
Comment (DiegoAlbertoTorres) [inline build/README.md]: Plx remove all files inside build/

**Comment by rinchenlama0075** [comment]
@DiegoAlbertoTorres, does this PR need to be closed before I can make a new one here from the same branch?

File: _includes/posts.html [hunk 5]
```diff
@@ -1,37 +1,54 @@
+<!-- switching from a bunch of if statement to elsif statement kept giving error:
```
Comment (DiegoAlbertoTorres) [inline _includes/posts.html:1]: Plx remove comment since we figured it out!

**Pull request closed without merging**